ironfish/src/rpc/routes/chain/getTransaction.ts

```diff
@@ -12,10 +12,14 @@ export type GetTransactionRequest = { blockHash: string; transactionHash: string
 export type GetTransactionResponse = {
   fee: string
   expiration: number
+  initialNoteIndex: number
   notesCount: number
   spendsCount: number
   signature: string
-  notesEncrypted: string[]
+  notesEncrypted: {
+    noteIndex: number
+    noteData: string
+  }[]
```

Comment:
This shouldnt be specified, normally you re-calculate this if your fetching the root noteSize